Translate to traditional Chinese (Taiwan)

tom83615 opened the pull request:
+ /config/i18n/zh_TW.json : traditional Chinese version file.

After creating your pull request, tick these boxes if they are applicable to you.

- [x] I have tested my changes against the `review` branch (the latest developmental version), and this pull request is targeting that branch as a base
- [x] I have tested my changes on Python 3.5/3.6

----

### Description

Add a langue file in i18n.

### Related issues (if applicable)

tom83615 committed:
Localization to Taiwan (traditional Chinese)

+ /config/i18n/zh_TW.json : traditional Chinese version file.

MattBSG reviewed (dismissed):
Update strings to current en counterparts

tom83615 committed:
Update zh_TW.json

update a new added strings from 
[Option to check if owner is in a server and leave if they're not. (#1597)]
